starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -686,6 +686,33 @@ sub check_dither {
         }
     }
 
+    # Small dither check for with creep-away
```

Comment:
Move this comment to where it actually applies, probably before line 698 or 703. But also make it more useful, again emphasizing *why* not *what* in code comments. Imagine me reading this in two years, so either provide the background or at least a link to the PR which has at least some hint of why.